core/nodepool/config/config.go

```diff
@@ -115,6 +117,7 @@ func (c ProvidedConfig) StackConfig(opts StackTemplateOptions) (*StackConfig, er
 
 	baseS3URI := strings.TrimSuffix(opts.S3URI, "/")
 	stackConfig.S3URI = fmt.Sprintf("%s/kube-aws/clusters/%s/exported/stacks", baseS3URI, c.ClusterName)
+	stackConfig.KubeResourcesS3Path = fmt.Sprintf("%sclusterBackup", strings.TrimPrefix(opts.S3URI, "s3://"))
```

Comment:
Are you willing to restore cluster(s) named differently from the original one or not?
If it was latter, it would be better to make KubeResourcesS3Path to be scoped like:
```
fmt.Sprintf("%s/kube-aws/clusters/%s/backup", strings.TrimPrefix(baseS3URI "s3://"), c.ClusterName)
```